Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -65,17 +65,14 @@ public protocol AppDelegateViewModelInputs {
   func didAcceptReceivingRemoteNotifications()
 
   /// Call when the app delegate receives a remote notification.
-  func didReceive(remoteNotification notification: [AnyHashable: Any], applicationIsActive: Bool)
+  func didReceive(remoteNotification notification: [AnyHashable: Any])
 
   /// Call when the app delegate gets notice of a successful notification registration.
   func didRegisterForRemoteNotifications(withDeviceTokenData data: Data)
 
   /// Call when the redirect URL has been found, see `findRedirectUrl` for more information.
   func foundRedirectUrl(_ url: URL)
 
-  /// Call when the user taps "OK" from the notification alert.
-  func openRemoteNotificationTappedOk()
```

Comment:
Has been removed from the protocol but still looks like we haven't removed this from `AppDelegateViewModel`

```
fileprivate let openRemoteNotificationTappedOkProperty = MutableProperty(())
public func openRemoteNotificationTappedOk() {
  self.openRemoteNotificationTappedOkProperty.value = ()
}
```

Reply:
Good catch thanks!